src/main/java/org/cryptomator/ipc/IpcMessageListener.java

```diff
@@ -5,10 +5,10 @@
 public interface IpcMessageListener {
 
 	default void handleMessage(IpcMessage message) {
-		if (message instanceof RevealRunningAppMessage) {
-			revealRunningApp();
-		} else if (message instanceof HandleLaunchArgsMessage m) {
-			handleLaunchArgs(m.args());
+		switch (message) {
+			case RevealRunningAppMessage x -> revealRunningApp();
+			case HandleLaunchArgsMessage hlam -> handleLaunchArgs(hlam.args());
+			default -> {}
```

Comment:
Since we can use preview features (and expect this to become stable with JDK 21), we can make `IpcMessage` a sealed class and remove the `default` branch.

Furthermore, let's stick to the convention of using single-letter pattern variables:

```suggestion
		switch (message) {
			case RevealRunningAppMessage m -> revealRunningApp(); // TODO: rename to _ with JEP 443
			case HandleLaunchArgsMessage m -> handleLaunchArgs(m.args());
```